pkgs/development/python-modules/python-fluent/fluent-pygments.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage, fetchFromGitHub, fluent-syntax, lib, pygments, pythonOlder, setuptools }:
+
+let
+  version = "1.0";
+  tag = "fluent.pygments@${version}";
+
+  src = fetchFromGitHub {
+    owner = "projectfluent";
+    repo = "python-fluent";
+    rev = tag;
+    hash = "sha256-AR2uce3HS1ELzpoHmx7F/5/nrL+7KhYemw/00nmvLik=";
+  };
+in
+buildPythonPackage {
+  pname = "fluent-pygments";
+  inherit version;
+  format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```

Also for others.

Reply:
I tried this, but I didn't get the build to succeed. Probably a skill issue, I'll try again.

Reply:
Maybe it doesn't work because it's a monorepo? I noticed that there is only one `pyproject.toml` that is shared across all three packages.

Reply:
Can you try to rm that top-level pyproject.toml in postPatch instead? then it should fall back.

Reply:
`pyproject.toml` doesn't exist in the release tag, only in the `main` branch.